docs/guides/client.md

```diff
@@ -0,0 +1,83 @@
+# Prism Client
+
+Prism inclues an Axios style HTTP Client that you can use to seamlessy perform requests to both a real server and a mocked document.
+
+### Create from filename or http resource
+
+```ts
+const client = await createClientFromResource('examples/petstore.oas2.yaml', {
+  mock: true,
+  validateRequest: true,
+  validateResponse: true,
+});
+```
+
+### Create from OAS document in string format
```

Comment:
```suggestion
### Create from OpenAPI string
```

Remember that we don't really talk about OAS for SEO reasons, and human readability. OpenAPI is a known brand, so whilst you might use oas for internal variables we generally want to say OpenAPI on anything user facing.